packages/analytics-js/src/components/eventManager/utilities.ts

```diff
@@ -53,11 +53,7 @@ const getUpdatedPageProperties = (
   properties: ApiObject,
   options?: Nullable<ApiOptions>,
 ): ApiObject => {
-  if (isUndefined(options?.page) || !isObjectLiteralAndNotNull((options as ApiOptions).page)) {
-    return mergeDeepRight(getContextPageProperties(), properties);
```

Comment:
why we changed logic here?

Reply:
The previous logic was deviating from v1.1 a bit.